apps/dashboard/app/controllers/batch_connect/sessions_controller.rb

```diff
@@ -16,15 +16,16 @@ def index
   # DELETE /batch_connect/sessions/1.json
   def destroy
     set_session
+    action_message_key = delete_session_panel? ? 'delete' : 'terminate'
 
-    if @session.destroy
+    if @session.destroy(delete_session_panel?)
```

Comment:
Instead of passing booleans we should probably just have a new method/api for deleting.

Passing booleans into a function to change it's behavior is something I've always heard not to do. So we likely should have a new `#terminate` on the controller and the session objects.

Reply:
OK - will update